frontend/projects/upgrade/src/app/core/experiments/store/experiments.model.ts

```diff
@@ -342,25 +350,49 @@ export interface ExperimentFormData {
 }
 
 export interface AddExperimentRequest {
```

Comment:
can we extend the DTO type so we get the type-safety w/o needing the notes? Or perhaps we need an experiment-stub type? Or "DraftExperimentDTO" type that has more optional fields than was previously needed? It would be great to improve our frontend-backend agreement on these experiment types, although that shouldn't hold up this PR.